scripts/scil_compute_fodf.py

```diff
@@ -17,15 +17,20 @@
 import argparse
 import logging
 
+from dipy.core.gradients import gradient_table
```

Comment:
The docstring above need updating no? 

Somehow, the --not_all part seems repetitive with the part above. No?

Please remove the and [Cote et al ...] ... (SDT). Pas rapport. Old and outdated